substrate-sgx/sp-io/src/lib.rs

```diff
@@ -101,6 +103,16 @@ pub enum KillStorageResult {
 	SomeRemaining(u32),
 }
 
```

Comment:
Do we need this?  I don't think we call any method that throws MultiRemovalResults. So we don't need the from transformation